tests/test_modernize.sh

```diff
@@ -6,6 +6,13 @@
 #
 # undo-modernizations.diff may need to be updated if new lines are added
 # that would be, but should not be modernized.
+
+PYVER=$(python --version 2>&1)
+[[ $PYVER =~ Python\ 2\. ]] || {
+    echo "Skipping modernize test on Python $PYVER"
```

Comment:
`$PYVER` already has "Python" in it, so does the echo string add a redundant Python?

Reply:
Thanks. Fixed in https://github.com/dimagi/commcare-cloud/pull/4222/commits/b69718700d00aa9d301a34530306b7761889326d